src/shipTemplate.h

```diff
@@ -118,6 +119,8 @@ class ShipTemplate : public PObject
     float jump_drive_max_distance;
     int weapon_storage[MW_Count];
 
+    int32_t science_database_entry = -1;
```

Comment:
`science_database_entry_id` would match the other code that stores a multiplayer ID instead of a direct link